frontend/src/pages/Fight.jsx

```diff
@@ -110,8 +115,10 @@ function Fight({ hero, bossesList }) {
           bossLife
         </progress>
 
-        <Jump spy={trigger}>
-          <Character character={currentBoss} className="fight-boss" />
+        <Jump spy={trigger} timeout={1000}>
```

Comment:
try:

    <Jump spy={currentBoss} timeout={1000}>

(and remove trigger state)